lib/widgets.php

```diff
@@ -11,6 +11,11 @@
  * @return boolean True if a screen containing the block editor is being loaded.
  */
 function gutenberg_is_block_editor() {
```

Comment:
Aside: This feels like it's not specific to widgets. Might make more sense surfaced up to the top-level `gutenberg.php` file. Then again, it's only necessary vs. `WP_Screen::is_block_editor` since the latter doesn't account for the widgets screen. I guess this is something we'll need to sort out sooner than later.

Reply:
It was kept on the widgets.php file temporarily to make merging Gutenberg to core easier (this file is not merged). The final solution should be updating WP_Screen::is_block_editor when things get more stable.